src/main/java/core/basesyntax/IsoscelesTrapezoid.java

```diff
@@ -0,0 +1,27 @@
+package core.basesyntax;
+
+public class IsoscelesTrapezoid extends Figure {
+    private static final int DIVIDER = 2;
```

Comment:
not necessary, geometry rules won't change

Reply:
i thought that it's magic number :)